### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: src/app/public/modules/list-view-grid/list-view-grid.component.html [hunk 1]
```diff
@@ -4,15 +4,14 @@
...
-    [height]="height | async"
+    [height]="gridHeight | async"
     [highlightText]="highlightSearchText ? (currentSearchText | async) : undefined"
     [rowHighlightedId]="rowHighlightedId"
     [selectedColumnIds]="selectedColumnIds | async"
     [selectedRowIds]="multiselectSelectedIds"
     [settingsKey]="settingsKey"
     [sortField]="sortField | async"
-    [width]="width | async"
+    [width]="gridWidth | async"
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/list-view-grid/list-view-grid.component.html:14]: Angular's template checker didn't like that we were putting a `number | Observable<number>` into an `async` pipe (only observables or promises are allowed). I had to use a separate public property that always returns an observable in the cases where a consumer provides only a number.

File: src/app/public/modules/list-view-grid/list-view-grid.component.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+:host {
+  ::ng-deep {
+    .sky-grid-heading {
+      border-top: 0 !important;
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/list-view-grid/list-view-grid.component.scss:4]: Somewhere along the line a border-top was added to the grid heading. Added this to get our visual tests to pass.

File: src/app/public/modules/list-view-grid/state/grid-state.state-node.ts [hunk 3]
```diff
@@ -9,6 +9,7 @@ import {
 
 @Injectable()
 export class GridState extends StateNode<GridStateModel> {
+  /* istanbul ignore next */
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/list-view-grid/state/grid-state.state-node.ts:12]: Istanbul doesn't like `super` calls.

File: src/app/public/modules/list-view-grid/list-view-grid.component.scss (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/list-view-grid/list-view-grid.component.scss:4]: I think this was from a recent toolbar fix with the search bar.

File: src/app/public/modules/list-view-grid/list-view-grid.component.ts [hunk 4]
```diff
@@ -130,7 +145,7 @@ export class SkyListViewGridComponent
 
   public currentSearchText: Observable<string>;
 
-  private multiselectSelectedIds: string[] = [];
+  public multiselectSelectedIds: string[] = [];
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/list-view-grid/list-view-grid.component.ts:148]: nice catch!

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**